Remotion/Validation/UnitTests/RuleBuilders/AddingPropertyValidationRuleBuilderTest.cs

```diff
@@ -97,58 +106,62 @@ public void AddMetaValidationRule_FuncOverload ()
     {
+      IPropertyMetaValidationRule actualRule = null;
       _propertyMetaValidationRuleCollectorMock
-          .Setup(
-              mock => mock.RegisterMetaValidationRule(
-                  It.Is<IPropertyMetaValidationRule>(
-                      rule => rule.GetType() == typeof(DelegatePropertyMetaValidationRule<IPropertyValidator>)
-                              && rule.Validate(new IPropertyValidator[0]).First().IsValid)))
-          .Verifiable();
+          .Setup(mock => mock.RegisterMetaValidationRule(It.IsAny<IPropertyMetaValidationRule>()))
+          .Callback((IPropertyMetaValidationRule r) => actualRule = r);
 
       _addingPropertyValidationBuilder.AddMetaValidationRule<IPropertyValidator>(v => true);
 
-      _propertyMetaValidationRuleCollectorMock.Verify();
+      Assert.That(actualRule, Is.TypeOf<DelegatePropertyMetaValidationRule<IPropertyValidator>>());
+      Assert.That(actualRule.Validate(new IPropertyValidator[0]), Has.Exactly(1).With.Property(nameof(MetaValidationRuleValidationResult.IsValid)).True);
     }
 
     [Test]
     public void AddMetaValidationRule_ExpressionOverload_IsValidIsFalse ()
     {
+      IPropertyMetaValidationRule actualRule = null;
       _propertyMetaValidationRuleCollectorMock
           .Setup(
-              mock => mock.RegisterMetaValidationRule(
-                  It.Is<IPropertyMetaValidationRule>(
-                      rule => rule.GetType() == typeof(DelegatePropertyMetaValidationRule<IPropertyValidator>)
-                              && !rule.Validate(new IPropertyValidator[0]).First().IsValid
-                              && rule.Validate(new IPropertyValidator[0]).First().Message
-                              == "Meta validation rule 'v => False' failed for validator 'Remotion.Validation.Validators.IPropertyValidator' "
-                              + "on property 'Remotion.Validation.UnitTests.TestDomain.Customer.UserName'.")))
-          .Verifiable();
+              mock => mock.RegisterMetaValidationRule(It.IsAny<IPropertyMetaValidationRule>()))
+          .Callback((IPropertyMetaValidationRule r) => actualRule = r);
 
       _addingPropertyValidationBuilder.AddMetaValidationRule<IPropertyValidator>(v => false);
 
-      _propertyMetaValidationRuleCollectorMock.Verify();
+      Assert.That(actualRule, Is.TypeOf<DelegatePropertyMetaValidationRule<IPropertyValidator>>());
+      Assert.That(actualRule.Validate(new IPropertyValidator[0]), Has.Exactly(1).Items);
+      Assert.That(actualRule.Validate(new IPropertyValidator[0]), Has.All.With.Property(nameof(MetaValidationRuleValidationResult.IsValid)).False);
+      Assert.That(actualRule.Validate(new IPropertyValidator[0]), Has.All.With.Property(nameof(MetaValidationRuleValidationResult.Message)).EqualTo(
+          "Meta validation rule 'v => False' failed for validator 'Remotion.Validation.Validators.IPropertyValidator' "
+          + "on property 'Remotion.Validation.UnitTests.TestDomain.Customer.UserName'."));
```

Comment:
```suggestion
      Assert.That(
          actualRule.Validate(new IPropertyValidator[0]),
          Has.All.With.Property(nameof(MetaValidationRuleValidationResult.Message))
              .EqualTo(
                  "Meta validation rule 'v => False' failed for validator 'Remotion.Validation.Validators.IPropertyValidator' "
                  + "on property 'Remotion.Validation.UnitTests.TestDomain.Customer.UserName'."));
```

Linebreaks: 
when a method call contains linebreaks, the arguments start in a new line.
For assertions, the Message-test (EqualTo etc) goes into a new line and depending on the length of the string, we can keep it in a signle line or need more line wraps.